code/game/objects/items/reagent_containers/food/snacks.dm

```diff
@@ -641,9 +641,19 @@
 /obj/item/reagent_container/food/snacks/carpmeat/Initialize()
 	. = ..()
 	reagents.add_reagent("fish", 3)
-	reagents.add_reagent("carpotoxin", 3)
+	reagents.add_reagent("carpotoxin", 6)
 	src.bitesize = 6
 
+/obj/item/reagent_container/food/snacks/carpmeat/processed
+	name = "processed carp fillet"
+	desc = "A fillet of spess carp meat. This one has been processed to remove carpotoxin."
+	icon_state = "fishfillet"
+	filling_color = "#FFDEFE"
+
+/obj/item/reagent_container/food/snacks/carpmeat/processed/Initialize()
```

Comment:
Since this is a subtype of `/obj/item/reagent_container/food/snacks/carpmeat`, and the `icon_state` and `filling_color` variables are the same here as on that, there's no need to redefine them. They'll be automatically "inherited" from the parent type.
```suggestion
/obj/item/reagent_container/food/snacks/carpmeat/processed
	name = "processed carp fillet"
	desc = "A fillet of spess carp meat. This one has been processed to remove carpotoxin."

/obj/item/reagent_container/food/snacks/carpmeat/processed/Initialize()
```

for reference:
https://github.com/cmss13-devs/cmss13/blob/7b583990b8d0be5676616726e8099e1e7b59dbf9/code/game/objects/items/reagent_containers/food/snacks.dm#L635-L639

Reply:
Thanks. Sorry, that should be obvious; my DM was very very rusty prior to working on this.